compiler/src/dmd/expressionsem.d

```diff
@@ -3729,12 +3729,13 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
             if (cd.isAbstract())
             {
                 exp.error("cannot create instance of abstract class `%s`", cd.toChars());
+                errorSupplemental(cd.loc, "class `%s` is located here", cd.toChars());
```

Comment:
how about _declared_ instead of located ?

Reply:
That sounds better